unittests/InstructionCountCI/Secondary.json

```diff
@@ -868,11 +868,12 @@
       ]
     },
     "packsswb mm0, mm0": {
-      "ExpectedInstructionCount": 4,
+      "ExpectedInstructionCount": 5,
       "Comment": "0x0f 0x63",
       "ExpectedArm64ASM": [
         "ldr d2, [x28, #768]",
-        "zip1 v2.2d, v2.2d, v2.2d",
+        "ldr d3, [x28, #768]",
```

Comment:
Looks like redundant MMX registers aren't tracked currently. Should be an easy fix.

Reply:
Thanks, will look into that

Reply:
MMX tracking added locally.